frontend/src/pages/Dashboard.js

```diff
@@ -84,23 +86,20 @@ const Dashboard = () => {
         updateAllTickets({ [TICKET_TYPES.UPR]: newUPRs })
     }
 
-    const getAllTickets = () => {
+    const getAllTickets = async () => {
         const endpoints = {
             [TICKET_TYPES.SF]: `${process.env.REACT_APP_BACKEND_URL}/sponsorshipfunds/`,
             [TICKET_TYPES.FI]: `${process.env.REACT_APP_BACKEND_URL}/fundingitems/`,
             [TICKET_TYPES.PPR]: `${process.env.REACT_APP_BACKEND_URL}/personalpurchases/`,
             [TICKET_TYPES.UPR]: `${process.env.REACT_APP_BACKEND_URL}/uwfinancepurchases/`,
         }
-        axios
-            .all(
-                Object.values(endpoints).map((endpoint) => axios.get(endpoint))
-            )
-            .then((responses) => {
-                Object.keys(endpoints).forEach((key, index) => {
-                    updateAllTickets({ [key]: responses[index].data })
-                })
-                console.log('fetched all tickets')
-            })
+        const data = await axios.all(
+            Object.values(endpoints).map((endpoint) => axios.get(endpoint))
```

Comment:
use axiosPreset and TICKET_ENDPOINTS to avoid hardcoding any endoints (should be able to remove `endpoints` const)